src/lxc/lxccontainer.h

```diff
@@ -841,6 +841,10 @@ struct bdev_specs {
 		char *lv; /*!< LVM Logical Volume name */
 		char *thinpool; /*!< LVM thin pool to use, if any */
 	} lvm;
+	struct {
+		char *rbdname; /*!< RBD image name */
+		char *rbdpool; /*!< Ceph pool name */
+	} rbd;
```

Comment:
This here is an API break, only append to existing struct, never insert new members in the middle!